lightkurve/lightcurvefile.py

```diff
@@ -373,6 +394,9 @@ def __init__(self, path, quality_bitmask='default', **kwargs):
         elif filetype is None:
             warnings.warn("Given fits file not recognized as Kepler or TESS "
                           "observation.", LightkurveWarning)
+        elif filetype == 'TessLightCurveFile-interact':
+            warnings.warn("Custom-made TESS fits files have limited "\
+                          "functionality.", LightkurveWarning)
```

Comment:
I think we'll want to get rid of this warning as well to avoid warning-fatigue (=> we don't want people to think that ignoring warnings is OK, so we should use them sparingly).